src/applications/vaos/appointment-list/components/RequestedAppointmentDetailsPage.jsx

```diff
@@ -37,6 +37,8 @@ const TIME_TEXT = {
 };
 
 export default function RequestedAppointmentDetailsPage() {
+  const queryParams = new URLSearchParams(window.location.search);
```

Comment:
I think we need to import this so it works in IE: `import URLSearchParams from 'url-search-params'`. I don't think we polyfilled this automatically, at least based on how others on vets-website are using it.

Reply:
I was going to ask you about this one, I saw the polyfill problem and was curious how we handled that here. 👍